src/sai/i2s.rs

```diff
@@ -584,14 +599,17 @@ macro_rules! i2s {
 }
 
 i2s! {
-    SAI2, Sai2: [i2s_sai1_ch_a, i2s_sai1_ch_b],
-    SAI3, Sai3: [i2s_sai2_ch_a, i2s_sai2_ch_b],
-    SAI1, Sai1: [i2s_sai3_ch_a, i2s_sai3_ch_b],
+    SAI1, Sai1: [i2s_sai1_ch_a, i2s_sai1_ch_b],
```

Comment:
@mtthw-meyer I guess the mismatch between the peripheral number and method name number is a just mistake. If so I'll just fix here.